wiki/en/Privacy-Statement.md

```diff
@@ -7,32 +7,38 @@ permalink: "/wiki/Privacy-Statement"
 
 # Privacy Statement
 
+## Definition of Terms
+
+- "**Server**" The Jamulus server software, as opposed to the host machine/OS it is running on.
+- "**Client**" The Jamulus software used to connect to a Server
+- "**Directory**" A Jamulus Server configured to display a list of Servers to Clients
+
 ## Web site
 
 The website at jamulus.io is served using GitHub Pages. We do not collect your personal data or set tracking cookies. When you connect to the jamulus.io site, your IP is sent to GitHub Pages.
```

Comment:
Should `jamulus.io` be a link?  Should we supply a link to Github's information about Github Pages?